server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_2__alter_connect_connection.sql

```diff
@@ -0,0 +1,4 @@
+--
+-- Add column `database_name` to `connect_connection` table
+--
+alter table `connect_connection` add column `database_name` varchar(255) DEFAULT NULL COMMENT 'Parameters which represent an instance of a database';
```

Comment:
better catalog_name than database_name,

catalog is a common concept in database area.

database_name may confused with other concept.


and postgres also use catalog_name in dictionary table.

Reply:
The databaseName field has been changed to catalogName